cmd/annotations_prep/main.go

```diff
@@ -61,6 +62,30 @@ func (s FeatureStatus) String() string {
 	return "Unknown"
 }
 
+// Source of the {{ .Collection.NameLowercase }}.
+type Source int
+
+const (
+	// Config indicates the source of the {{ .Collection.NameLowercase }} is
+	// user configuration.
+	Config Source = iota
+
+	// Runtime indicates the source of the {{ .Collection.NameLowercase }} is
+	// the Istio runtime (i.e. generated by Istio).
+	Runtime
```

Comment:
Many features are set to default values by Istio, but can be overridden by users.  In that case, is the Source Runtime or Config?